<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1323</width>
    <height>713</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ManaR</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>40</y>
      <width>286</width>
      <height>461</height>
     </rect>
    </property>
    <property name="title">
     <string>Card view</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QGraphicsView" name="graphicsView_card"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="pushButton_prev">
         <property name="text">
          <string>&lt;--</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_next">
         <property name="text">
          <string>--&gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>ManaRlize</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>540</y>
      <width>590</width>
      <height>127</height>
     </rect>
    </property>
    <property name="title">
     <string>Deck Statistics</string>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QTextEdit" name="textEdit"/>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>60</y>
      <width>286</width>
      <height>275</height>
     </rect>
    </property>
    <property name="title">
     <string>Mana Curve</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGraphicsView" name="graphicsView_mana"/>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Update</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_base">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>281</width>
      <height>501</height>
     </rect>
    </property>
    <property name="title">
     <string>Base Deck</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QListView" name="listView_base"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_type_base">
         <property name="text">
          <string>Type</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_type_base">
         <item>
          <property name="text">
           <string>All Cards</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Artifact</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Creature</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Enchantment</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Instant</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Land</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Planeswalker</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Sorcery</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QCheckBox" name="checkBox_r_base">
         <property name="styleSheet">
          <string notr="true">background-color:rgb(128, 0, 0)</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_g_base">
         <property name="styleSheet">
          <string notr="true">background-color:rgb(0, 128, 0)</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_w_base">
         <property name="styleSheet">
          <string notr="true">background-color:rgb(255, 255, 255)</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_u_base">
         <property name="styleSheet">
          <string notr="true">background-color:rgb(0, 0, 255)</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_b_base">
         <property name="styleSheet">
          <string notr="true">background-color:rgb(0, 0, 0)</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="labe_mana_base">
         <property name="text">
          <string>Mana Cost</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_mana_base">
         <item>
          <property name="text">
           <string>&gt;=</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>=</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>&lt;</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>&lt;=</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>&gt;</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBox_mana_base">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>50</number>
         </property>
         <property name="value">
          <number>1</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Ability</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox">
         <item>
          <property name="text">
           <string>Any</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Dearthtouch</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Flash</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Flying</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Haste</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Hexproof</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Indestructible</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Lifelink</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Morph</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Outlast</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Protection</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Reach</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Vigilance</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <zorder>groupBox</zorder>
   <zorder>groupBox_3</zorder>
   <zorder>groupBox_3</zorder>
   <zorder>groupBox_2</zorder>
   <zorder>comboBox</zorder>
   <zorder>label</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1323</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDeck">
    <property name="title">
     <string>Deck</string>
    </property>
    <addaction name="actionLoad_Deck"/>
    <addaction name="actionSave_Deck"/>
    <addaction name="separator"/>
    <addaction name="actionDownload_Images"/>
    <addaction name="separator"/>
    <addaction name="actionManaRlize"/>
    <addaction name="actionUpdate_Mana_Curve"/>
    <addaction name="actionGet_Statistics"/>
   </widget>
   <widget class="QMenu" name="menuCard">
    <property name="title">
     <string>Card</string>
    </property>
    <addaction name="actionPrint_URL"/>
    <addaction name="actionPrint_JSON"/>
    <addaction name="separator"/>
    <addaction name="actionNext_Card_in_Deck"/>
    <addaction name="actionPrev_Card_in_Deck"/>
   </widget>
   <addaction name="menuDeck"/>
   <addaction name="menuCard"/>
  </widget>
  <action name="actionLoad_data_base">
   <property name="text">
    <string>Load data base</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionLoad_Deck">
   <property name="text">
    <string>Load Deck</string>
   </property>
  </action>
  <action name="actionSave_Deck">
   <property name="text">
    <string>Save Deck</string>
   </property>
  </action>
  <action name="actionDownload_images">
   <property name="text">
    <string>Download images</string>
   </property>
  </action>
  <action name="actionPrint_URL">
   <property name="text">
    <string>Print URL</string>
   </property>
  </action>
  <action name="actionManaRlize">
   <property name="text">
    <string>ManaRlize</string>
   </property>
  </action>
  <action name="actionGet_Statistics">
   <property name="text">
    <string>Get Statistics</string>
   </property>
  </action>
  <action name="actionUpdate_Mana_Curve">
   <property name="text">
    <string>Update Mana Curve</string>
   </property>
  </action>
  <action name="actionExit_2">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionPrint_JSON">
   <property name="text">
    <string>Print JSON</string>
   </property>
  </action>
  <action name="actionNext_Card_in_Deck">
   <property name="text">
    <string>Next Card in Deck</string>
   </property>
  </action>
  <action name="actionPrev_Card_in_Deck">
   <property name="text">
    <string>Prev Card in Deck</string>
   </property>
  </action>
  <action name="actionDownload_Images">
   <property name="text">
    <string>Download Images</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
